doc/src/learn/why-move.md

```diff
@@ -2,7 +2,21 @@
 title: Why Move?
 ---
 
-This page compares the Move and Solidity programming languages For a full description of the issues with traditional smart contract languages, see the [Move Problem Statement](https://github.com/MystenLabs/awesome-move/blob/main/docs/problem_statement.md).
+This page links to key Move resources and compares the Move and Solidity programming languages For a full description of the issues with traditional smart contract languages, see the [Move Problem Statement](https://github.com/MystenLabs/awesome-move/blob/main/docs/problem_statement.md).
```

Comment:
Missing a period between "Solidity programming languages" and "For a full description"

Reply:
> Missing a period between "Solidity programming languages" and "For a full description"

Great catch! Fixing.